code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -756,6 +756,68 @@ datum
 					holder.my_atom.color = "#ffffff"
 				return ..()
 
+		harmful/super_ipecac // vomit your heart out
+			name = "super ipecac"
```

Comment:
I agree it should be a daft fantasy name because of the effect, but I'm not a huge fan of having variants of normal chems with wildly different effects. Maybe just something like "hyper vomitium"?